### DISCUSSION TITLE
Failure to read attributes of vault.cryptomator.

### DISCUSSION THREAD

**Discussion by Adhambec** [iterate-ch/cyberduck#14957] [Q&A] [answered]
Failure to read attributes of vault.cryptomator.
Software caused connection abort: socket write error. The connection attempt was rejected. The server may be down, or your network may not be properly configured.
```
MLST/www/mysite.uz/public/vault.cryptomator
421 Idle timeout (600 seconds): closing control connection
```
os: windows 10
cyberduck Version: 8.6.2

**Comment by dkocher** [comment]
Can you give some more context about when you encounter this error and ideally give us the steps required to reproduce.

**Comment by Adhambec** [comment]
> Can you give some more context about when you encounter this error and ideally give us the steps required to reproduce.
I think 600 seconds after connecting to the server

**Comment by SkyMatt70** [comment]
Getting the same on 8.6.2 Mac.  Seems when trying to save a file from an external editor that I've been working on for more than 10 minutes.  Spinner at the bottom with "Prepare <myfilename> (Overwrite)" just sits for a while then get the popup.

Try Again fails with the same popup.  Restarting Cyberduck is the only fix and then it's lost track of my file.

**Comment by dkocher** [comment, marked as answer]
Similar to #13728. Disabling _Preferences → Cryptomator → Auto detect and open vault in browser_ will probably display a slightly different error message but the underlying cause seems to be the idle timeout on the control connection to the server. You should be able to resolve this by enabling _Preferences → Connection → Repeat failed networking tasks_.
